invoices/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "@gristlabs/widget-invoices",
+  "description": "invoices",
```

Comment:
I think you may have cornered yourself into volunteering to write one-line descriptions of these widgets :)

Reply:
:)